### ISSUE TITLE
Refactor cable cutting interaction out of InteractableTiles

### ISSUE THREAD

**Issue by NoooneyDude** [unitystation/unitystation#6497]
For whatever reason, the code responsible for displaying the cable cutting window for the client resides in the main `InteractableTiles` class. This class is responsible for delegating out interactions to the tile, so that each tile can determine the interaction.

I don't see any reason why the cable cutting window should be unique in this way, so it should be refactored into its own interactable tile class, deriving from `TileInteraction` and added to the tile SO's interactions list. As the UI is only needed client-side only, can probably just let `WillInteract()` return false once done with the window spawn.

It might also make sense to merge `LoadCableCuttingWindow` into this new interaction file.

**Comment by Korywon** [comment]
Good idea. That would help decouple the cable cutting window logic. I'll take a stab at this one, and try to also move the cable cutting window into the UI manager, rather than use `Resources.Load()`.
